homeassistant/components/homematicip_cloud/device.py

```diff
@@ -76,8 +76,16 @@ def device_info(self):
 
     def _async_device_changed(self, *args, **kwargs):
         """Handle device state changes."""
-        _LOGGER.debug("Event %s (%s)", self.name, self._device.modelType)
-        self.async_schedule_update_ha_state()
+        # Don't update disabled entities
+        if self.registry_entry is None or not self.registry_entry.disabled:
```

Comment:
Feel free to add this snippet to entity.py

    @property
    def enabled(self):
        """Return if the entity is enabled in the entity registry."""
        return self.registry_entry is None or not self.registry_entry.disabled

Reply:
Thanks.